core/store/fixtures/fixtures.sql

```diff
@@ -1,9 +1,35 @@
-INSERT INTO users (email, hashed_password, token_hashed_secret, created_at, updated_at) VALUES (
+INSERT INTO users (email, hashed_password, token_hashed_secret, role, created_at, updated_at) VALUES
+(
```

Comment:
Ah okay - I've updated my new tests to create (and drop) temporary test users that have the correct role I need instead of added more static test users to the fixtures here.

Reply:
Nice work, this is much better